src/modules/bulletproofs/bulletproofs_pp_norm_product_impl.h

```diff
@@ -77,4 +77,67 @@ static void secp256k1_bulletproofs_powers_of_r(secp256k1_scalar *powers, const s
         secp256k1_scalar_sqr(&powers[i], &powers[i - 1]);
     }
 }
+
+typedef struct ec_mult_bp_commit_cb_data {
```

Comment:
This is a really a micro nit: We usually spell this `ecmult` in code/identifiers (instead of `ec_mult`)